x-pack/plugins/ml/public/application/components/job_messages/job_messages.tsx

```diff
@@ -21,14 +28,21 @@ interface JobMessagesProps {
   loading: boolean;
   error: string;
   refreshMessage?: React.MouseEventHandler<HTMLButtonElement>;
+  actionHandler?: (message: JobMessage) => void;
 }
 
 /**
  * Component for rendering job messages for anomaly detection
  * and data frame analytics jobs.
  */
-export const JobMessages: FC<JobMessagesProps> = ({ messages, loading, error, refreshMessage }) => {
-  const columns = [
+export const JobMessages: FC<JobMessagesProps> = ({
+  messages,
+  loading,
+  error,
+  refreshMessage,
+  actionHandler,
+}) => {
+  const columns: Array<EuiBasicTableColumn<any>> = [
```

Comment:
Should this be `Array<EuiBasicTableColumn<JobMessage>>`?

Reply:
Updated in 7a3786bd40c401912ab742b4a05bc13d80990058